packages/express/src/expressCollector.ts

```diff
@@ -41,7 +41,7 @@ export class ExpressCollector<
 
   public createErrorHandler(): ErrorRequestHandler {
     return (err, req, res, next) => {
-      this.logger.log(LogLevel.ERROR, err);
+      this.logger.error(err);
```

Comment:
1. why? 2. Co toto taky z options? Uplne stejne jako na radku 148

Reply:
nope - tohle neni HTTP error, tohle je uncaught application error, nema to bejt stejnej level jako 5xx nebo 4xx response (a po tomhle se stejne zaloguje response s tim nastavenym `errorLevel`)

Reply:
ta zmena na tomhle radku je jen proto, ze je to posledni misto v tom souboru ktery pouzivalo `LogLevel`, takhle to ma o import min a funguje to stejne

Reply:
1. ok, 2. customibilita?

Reply:
je to vubec slovo? :D

Reply:
"customisability" je podle me funkcni slovo.. ale v tomhle pripade bych to klidne nechal takhle, imo prilis mnoho options skodi ;-) nebo jakoby nevim, jenom mi prijde ze tohle zrovna klidne muze bejt `ERROR` proste natvrdo..

Reply:
tak treba toto taky na tetrisu hazu do vlastniho levelu...

Reply:
better now?